sujeong/app.js

```diff
@@ -0,0 +1,64 @@
+const http = require('http');
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const dotenv = require('dotenv');
+
+//모든 RES에 CORS요청을 설정하는법 
+// app.use(cors())
+// app.get('/ping', function (req, res, next){
+//     res.json({message:'pong'})
+// })
+// app.listen(3000, function(){
+//     console.log('server listening on port 3000')
+// })
+
+dotenv.config() //환경변수보다 위에 놓기
+
```

Comment:
dotenv는 최상단에 아래와 같이 작성하여 수정해 주세요 !
```suggestion
require('dotenv').config() 
```